includes/admin/helpers/class-rop-cron-helper.php

```diff
@@ -204,8 +204,30 @@ public function remove_cron( $request = array() ) {
 		 */
 		$this->cron_status_global_change( false );
 
+		$rop_cron_hooks = array( self::CRON_NAMESPACE, self::CRON_NAMESPACE_ONCE );
+
+		if ( function_exists( 'as_get_scheduled_actions' ) ) {
+			foreach ( $rop_cron_hooks as $cron_hook ) {
```

Comment:
can't we abstract this aswell?

Reply:
@selul I've made it an abstract method. Please check the latest commit.